packages/now-ruby/index.ts

```diff
@@ -0,0 +1,223 @@
+import { join, dirname } from 'path';
+import execa from 'execa';
+import { ensureDir, move, remove, stat, readFile, writeFile } from 'fs-extra';
+import mm from 'micromatch';
```

Comment:
```suggestion
import { glob } from '@now/build-utils';
```

Let's use glob instead since `includeFiles` can be a glob

Reply:
Actually you can remove this line and the `micromatch` dependency since glob is imported below.

Reply:
@nathancahill Can `micromatch` be removed?